.github/workflows/build.yml

```
@@ -19,4 +19,86 @@
+            parameters = $parametersJson
+        }
+
+        # Convert the body to JSON
+        $bodyJson = $body | ConvertTo-Json
+
+        # Define the URL
+        $url = 'https://dev.azure.com/' + $organization + '/' + $project + '/_apis/pipelines/' + $pipelineId + '/runs?api-version=7.1'
+
+        # Trigger the pipeline
+        Invoke-RestMethod -Uri $url -Method Post -Headers $headers -Body $bodyJson
+        "
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)